test/15-conversions.spec.coffee

```diff
@@ -36,14 +36,3 @@ describe 'conversions', ->
 			expect(conversion.envArrayToObject('')).to.deep.equal({})
 			expect(conversion.envArrayToObject([])).to.deep.equal({})
 			expect(conversion.envArrayToObject(1)).to.deep.equal({})
-
-		it 'should ignore leading and trailing whitespace', ->
```

Comment:
Can you change this to test that leading/trailing whitespace is handled correctly please